docs/concepts/configuration/pod-priority-preemption.md

```diff
@@ -0,0 +1,177 @@
+---
+approvers:
+- davidopp
+- wojtek-t
+title: Pod Priority and Preemption
+---
+
+[Pods](/docs/user-guide/pods) in Kubernetes 1.8 and later can have priority. Priority
+indicates the importance of a pod. When a pod cannot be scheduled, scheduler tries
+to preempt lower priority pods in order to make scheduling of the pending pod possible.
+
+* TOC
+{:toc}
+
+## How to use it
+In order to use priority and preemption in Kubernetes 1.8, you should follow these
+steps:
+
+1. Enable Priority and Preemption.
+1. Add one or more PriorityClasses.
+1. Create pods with `PriorityClassName` set to one of the added PriorityClasses.
+
+The following sections provide more information about these steps.
+
+## Enable Priority and Preemption
+Pod priority and preemption is disabled by default in Kubernetes 1.8 as it is an
+__alpha__ feature. It can be enabled by a command-line flag:
+
+```
+--feature-gates=PodPriority=true
+```
+
+Once enabled you can add PriorityClasses and create pods with `PriorityClassName` set.
+If you tried it and decided to disable it, you must remove this command-line flag or
+set it to false and restart API server and Scheduler. Once disabled, the existing
+pods may keep their priority fields, but preemption will be disabled and priority
+fields will be ignored.
+
+**Note:** Alpha features should not be used in production systems! Alpha 
+features are more likely to have bugs and future changes to them are not guaranteed to
+be backward compatible.
+
+## PriorityClass
+A PriorityClass object defines a mapping from a PriorityClassName to the integer
+value of the priority. The higher the value, the higher the priority. PriorityClass
+objects can have any 32-bit integer value smaller than or equal to 1 billion. Larger
+numbers are reserved for system use.
+
+PriorityClass also has two optional fields: `globaleDefault` and `description`.
```

Comment:
globaleDefault -> globalDefault

Reply:
For symmetry, I would suggest to say the names of the required fields (`name` and `value`).